api/src/controllers/plantController.js

```diff
@@ -47,7 +53,7 @@ export const getLimitedTopRatePlantsCard = async (req, res) => {
       return res.status(400).json({ message: "Invalid limit number" });
     }
     const result = await db("plants")
-      .select("id", "name" , "description", "price", "image_url")
+      .select("id", "name", "description", "price", "image_url")
       .orderBy("average_rating", "asc")

```

Comment:
Plants are ordered by "average_rating" in ascending order, which will return the lowest-rated plants first. If the intention is to get top-rated plants, use "desc" instead of "asc" for the order direction.